syft/frameworks/torch/mpc/przs.py

```diff
@@ -0,0 +1,55 @@
+from collections import defaultdict
+
+import torch
+
+import syft
+from syft.generic.utils import allow_command, remote
+
+RING_SIZE = 2 ** 64
+
+
+def przs_setup(workers):
+    seed_max = 2 ** 32
+    cycle_workers = zip(workers, [*workers[1:], workers[0]])
```

Comment:
```suggestion
    shifted_workers = [workers.pop()]+[workers]
    cycle_workers = zip(workers, shifted_workers)
```
for readability, the name "cycle workers" also need to be changed, I don't have a better name but the reader should understand that it's a list of (workers[i], workers [i+1]) cycle workers doesn't convey that meaning